src/lib/select/select.spec.ts

```diff
@@ -1362,7 +1369,8 @@ describe('MdSelect', () => {
   selector: 'basic-select',
   template: `
     <div [style.height.px]="heightAbove"></div>
-    <md-select placeholder="Food" [formControl]="control" [required]="isRequired">
+    <md-select placeholder="Food" [formControl]="control" [required]="isRequired"
+      [tabIndex]="tabIndexOverride">
```

Comment:
I went this route to be consistent with the other modules (e.g. the checkbox). IIRC we named it `tabIndex`, because the property on the native DOM element is called `tabIndex`.